config/packages/security.yaml

```diff
@@ -35,6 +38,8 @@ security:
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
     access_control:
-        - { path: ^/login$, roles: IS_AUTHENTICATED_ANONYMOUSLY }
```

Comment:
It probably a good idea to have this line because it ensure that the login page is accessible to anyone, which must obviously be the case.